client/dashboard/plugins/manage/components/action-render-modal.tsx

```diff
@@ -65,20 +68,22 @@ function getConfirmText( actionId: string, items: PluginListRow[] ) {
 	if ( items.length === 1 ) {
 		const pluginName = items[ 0 ].name;
 		const count = items[ 0 ].sitesCount;
+		const activeCount = items[ 0 ].sitesWithPluginActive.length;
+		const inactiveCount = items[ 0 ].sitesWithPluginInactive.length;
 		switch ( actionId ) {
 			case 'activate':
 				return sprintf(
 					// translators: %1$s is the plugin name. %2$d is the number of sites.
-					__( 'You are about to activate the %1$s plugin installed on %2$d sites.' ),
+					__( 'You are about to activate the %1$s plugin on %2$d sites.' ),
```

Comment:
Question: Should this use _n as well to take into account 1 site?

Reply:
Better yes, thanks for pointing it out.